manuscript/0-introduction.txt

```diff
@@ -1,61 +1,61 @@
-# Introduction
+# Введение
 
-## Background
+## Предыстория
 
-The PHP community is in an interesting phase right now and it has for a long time lagged behind on the more traditional software development practices. Practices that have been more or less considered as given in other communities. Continuous integration, package management, dependency injection and adopting object oriented programming to its extent to name a few. But as of PHP 5.3 there are no more excuses to why you can't write modern and clean code. The community have responded to this and a great number of people have stepped forward in supporting this through teaching and building tools for accomplishing these practices.
```

Comment:
The PHP community is in an interesting phase right now, and it has been lagging for a long time behind the more traditional software development practices - the ones that have been considered more or less as a standard in other communities. Continuous integration, package management, adopting dependency injection and other object-oriented programming principles — are just a few examples to mention. But as of PHP 5.3, there are no more excuses for why you can't write modern and clean code. The community has responded to this, and a significant number of people have stepped forward in supporting this through teaching and building tools for accomplishing these practices.